fix(button,statuslight,picker,etc): address regressions in migrated components (#3687)
adobe/spectrum-css: .changeset/breezy-impalas-push.md

---
"@spectrum-css/switch": patch
---

> ### S2 switch fixes

Reviewer: You did this so fast!

> This work addresses some very minor regressions noticed in the migrated S2 switch. We have reimplemented the switch animation durations tokens, and updated their usage within the style definitions correctly.
> 
> PRs used as reference:
> 
> - [S2 switch migration](https://github.com/adobe/spectrum-css/pull/2651)

Reviewer: ```suggestion
This work addresses minor regressions in the Switch component. Animation tokens are reimplemented and their usage updated within the style definitions.

Reference:

- [S2 switch migration](https://github.com/adobe/spectrum-css/pull/2651)
```

Just a little wordsmithing nit on my part. Looks great!
